notebooks/202-vision-superresolution/202-vision-superresolution-video.ipynb

```diff
@@ -329,15 +325,15 @@
     "comparison_video_path = video_path.with_name(\n",
     "    f\"{video_path.stem}_superres_comparison.mp4\"\n",
```

Comment:
The output videos in this notebook are saved in the same directory as the source video. They should be downloaded to the output directory, following the new structure.